en/gps_compass/README.md

```diff
@@ -22,55 +17,66 @@ The internal compass *may* be useful on larger vehicles (e.g. VTOL) where it is
 On small vehicles an external compass is almost always required.
 :::
 
-## Combined GPS/Compass Options
+## Supported GPS and/or Compass
+
+Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [Yaw Output](#configuring-gps-as-yaw-heading-source) | [Dual FP9 GPS Heading](../gps_compass/u-blox_f9p_heading.md)
+:--- | :---: | :---:  | :---:  | :---:  | :---: | :---:
+[Hobbyking u-blox Neo-M8N GPS with Compass](https://hobbyking.com/en_us/ublox-neo-m8n-gps-with-compass.html?gclid=Cj0KCQjwqM3VBRCwARIsAKcekb3ojv1ZhLz1-GuvCsUuGT8ZZuw8meMIV_I6pgUCj6DJRzHBY9OApekaAgI5EALw_wcB&gclsrc=aw.ds&___store=en_us) | Neo-M8N | &check; | | |
```

Comment:
Some refer to Neo-8MN, which I assume is the "proper name", while others refer to 8MN. OK if I use 8MN everywhere?